build.gradle

```diff
@@ -76,7 +76,7 @@ ext {
     isMasterBranch = gitBranch == "master"
     isDevelopBranch = gitBranch == "develop"
     canUploadArchives = isMasterBranch || isDevelopBranch
-    isSnapshotVersion = project.version.contains("SNAPSHOT")
+    isSnapshotVersion = project.version.contains("NEXT")
```

Comment:
Should we keep `isSnapshotVersion` name? We could probably rename it to `isNextVersion`.

Reply:
I found `isSnapshotVersion` more intuitive because if it is the case we push to the snapshots repo of nexus https://github.com/iExecBlockchainComputing/iexec-common/blob/develop/build.gradle#L114